submissions/SofiiaTrokhymchuk/pop-up-task/css/style.css

```diff
@@ -0,0 +1,217 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    text-decoration: none;
+    list-style-type: none;
+}
+
+body {
+    font-family: Verdana, Tahoma, sans-serif;
+    color: #212121;
+    background: #e6e6e6;
+    font-size: 1.2rem;
+    height: 100vh;
+}
+
+a:link,
+a:visited {
+    color: inherit;
+}
+
+img[src$=".svg"] {
+    max-height: 30px;
+}
+
+.container {
+    padding: 15px 50px;
+    max-width: 1000px;
+    margin: 0 auto;
+    position: relative;
+}
+
+.header {
+    display: grid;
+    align-items: center;
+    grid-template-columns: 1fr 2fr;
+    padding: 15px 50px;
+    margin: -15px -50px;
+    margin-bottom: 0;
+    box-shadow: 0 7px 15px -15px #4f4f4f;
+}
+
+.header-menu {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.header-menu__item a,
+.label-wrapper {
+    display: block;
+    padding: 7px;
+    cursor: pointer;
+    font-weight: bold;
+}
+
+.header-menu__item>a:hover,
+.header-menu__item>a:focus {
+    background-color: #212121;
+    color: #e6e6e6;
+    border-radius: 5px;
+}
+
+.header-menu__item>a:active {
+    color: #f3f3f3;
+}
+
+#pop-up-checkbox:focus~label>div,
```

Comment:
better to use spaces for better readability `#pop-up-checkbox: focus ~ label > div`